File: app/src/main/java/com/song/deviceinfo/ui/debug/DebugViewModel.java
package com.song.deviceinfo.ui.debug;

import android.content.Context;

import com.song.deviceinfo.info.DebugInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/7/14.
 */
public class DebugViewModel extends NormalViewModel {

    public List<Pair<String, String>> getDebugInfo(Context context) {
        return DebugInfo.getDebugInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/debug/DebugViewModel.kt
package com.song.deviceinfo.ui.debug

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.DebugInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/7/14.
 */
class DebugViewModel : NormalViewModel() {
    fun getDebugInfo(context: Context?): List<Pair<String, String>> {
        return DebugInfo.getDebugInfo(context!!)
    }
}
